### PR TITLE
Add fixtures for shipping method rules

### PR DESCRIPTION
**Pull request by loevgaard**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

Adds fixtures for shipping method rules

### CODE REVIEW

**Review by Zales0123 — APPROVED** [review]

File: src/Sylius/Component/Core/Factory/ShippingMethodRuleFactory.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    {
+        return $this->createPromotionRule(TotalWeightGreaterThanOrEqualRuleChecker::TYPE, ['weight' => $weight]);
+    }
+
+    public function createWeightLessThanOrEqual(int $weight): ShippingMethodRuleInterface
+    {
+        return $this->createPromotionRule(TotalWeightLessThanOrEqualRuleChecker::TYPE, ['weight' => $weight]);
+    }
+
+    private function createPromotionRule(string $type, array $configuration): ShippingMethodRuleInterface
```
Comment (lchrusciel) [inline src/Sylius/Component/Core/Factory/ShippingMethodRuleFactory.php:62]:
```suggestion
    private function createShippingMethodRule(string $type, array $configuration): ShippingMethodRuleInterface
```

File: src/Sylius/Component/Core/Factory/ShippingMethodRuleFactory.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    public function createWeightGreaterThanOrEqual(int $weight): ShippingMethodRuleInterface
+    {
+        return $this->createPromotionRule(TotalWeightGreaterThanOrEqualRuleChecker::TYPE, ['weight' => $weight]);
+    }
+
+    public function createWeightLessThanOrEqual(int $weight): ShippingMethodRuleInterface
+    {
+        return $this->createPromotionRule(TotalWeightLessThanOrEqualRuleChecker::TYPE, ['weight' => $weight]);
+    }
```
Comment (lchrusciel) [inline src/Sylius/Component/Core/Factory/ShippingMethodRuleFactory.php:60]: All these methods are not used nor tested(with unit test). For me, it is a dead code and should be removed (or tested)
```suggestion
```

Comment (Zales0123) [reply src/Sylius/Component/Core/Factory/ShippingMethodRuleFactory.php:60]: Oh yes, definitely we need to test it or to remove it 👍

**Comment by Rafikooo** [comment]
Hello Joachim, thank you for submitting this pull request.
Unfortunately, it has been some time since any updates have been made to the codebase, and there are now conflicts with the latest changes. Additionally, we have not received a response from you regarding the review comments that were left on the pull request. As a result, we will be closing this pull request in the near future, unless you still want to contribute. 
Best wishes 🚀

**Comment by Rafikooo** [comment]
Hello @loevgaard,
We have decided to close all fixtures-related PRs, as we plan to migrate to [zenstruck/foundry](https://github.com/Sylius/Sylius/issues/13621)
Best regards 🚀

**Comment by loevgaard** [comment]
That's a good idea :)

**Pull request closed without merging**